templates/claude/lib/knowledge_buffer.sh

```
@@ -7,8 +7,20 @@
 #
 # common.sh の vibecorp_repo_id / vibecorp_cache_root を再利用する。
 
+# zsh / bash 両対応のソースパス解決
+# - bash:  ${BASH_SOURCE[0]}
+# - zsh:   ${(%):-%x}（zsh のプロンプト展開フラグ %x = 現在 sourced 中のファイル名）
+# - その他: $0（POSIX フォールバック）
+# bash の if-false ブランチでは ${(%):-%x} は parse error にならず実害なし
+if [ -n "${BASH_VERSION:-}" ]; then
+  _knowledge_buffer_lib_src="${BASH_SOURCE[0]}"
+elif [ -n "${ZSH_VERSION:-}" ]; then
+  _knowledge_buffer_lib_src="${(%):-%x}"
```

Shellcheck (0.11.0)

[error] Parameter expansions can't start with (. Double check syntax.

(SC2296)